rollbar-java/src/main/java/com/rollbar/notifier/sender/SyncSender.java

```diff
@@ -71,9 +73,12 @@ private HttpURLConnection getConnection() throws IOException {
   }
 
   private void sendJson(HttpURLConnection connection, byte[] bytes) throws IOException {
-    try (OutputStream out = connection.getOutputStream()) {
+    OutputStream out = null;
+    try {
+      out = connection.getOutputStream();
       out.write(bytes, 0, bytes.length);
     } catch (IOException e) {
+      ObjectsUtils.close(out);
```

Comment:
This should be moved to a `finally` block, could you please change it?

Reply:
Yes, sure. My bad.